tests/test_generators/python/test_python_ifabsent_processor.py

```diff
@@ -403,7 +441,7 @@ def test_enum_default_value():
             schema_view.all_slots()[SlotDefinitionName("presence")],
             schema_view.all_classes()[ClassDefinitionName("Student")],
         )
-        == "PresenceEnum.Missing"
+        == "'PresenceEnum.Missing'"
```

Comment:
Great! I like the string approach more than putting `None` there :) and testing this out it looks like it's working.